virt/virt-4-10-release-notes.adoc

```diff
@@ -69,12 +69,10 @@ The SVVP Certification applies to:
 
 //CNV-11201 Rollout of NodeNetworkConfigurationPolicy
 
-//CNV-13660 Inherit static IP from a NIC attached to the bridge
 
 //CNV-13679 SR-IOV live-migration without privilege escalation
 * xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Live migration] is now supported by default for virtual machines that are attached to an SR-IOV network interface.
 
-//CNV-13680 Primary and secondary networks over a shared NIC on OVN clusters
```

Comment:
Seems that this removal is not related to the PR.

Reply:
Correct. This feature has been pushed to 4.11, so I just deleted the placeholder for the release note.